Makefile

```diff
@@ -0,0 +1,26 @@
+.PHONY: install run format lint
+
+install:
+    @pip install -r requirements.txt
+    @python -m pip install --upgrade pip
+    @echo "-dependencies installed"
+
+test:
+    @pip install pytest
```

Comment:
`pytest` will be installed with other requiements. This command is supposed to run tests instead